### PR TITLE
Fix: add message to progress dialogs

### LINKED ISSUE #1223: Some progress dialogues don't show a message
**Summary:** 
Some progress dialogues don't show a proper message, they just show this placeholder: 'message'.

**Steps to reproduce:** 
There are many steps to reproduce this issue since there are many progress dialogues that don't work but here's one:

Clients>Create client>submit

**Expected behaviour:** 
The app should show a proper progress dialogue message.

**Observed behaviour:** 
The app just shows a placeholder text ('message') instead of a proper message that actually describes what the user is waiting for.

**Device and Android version:** 
Model: Condor A8
Android version: 7.0

**Screenshots:** 
Screenshot_20191209-203441

### PR DESCRIPTION
**Pull request by OussEmaDevCode**
Fixes #1223 

Please Add Screenshots If there are any UI changes.

I updated all the progress dialogues not showing messages and here's a sample of them :

Screenshot_20191209-212142

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `MifosStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by OussEmaDevCode** [comment]
@elirehema, @moksh-mahajan @santoshconflux Hi : ), please have a look at this

File: mifosng-android/src/main/java/com/mifos/utils/SafeUIBlockingUtility.java [hunk 1]
```diff
@@ -36,6 +36,16 @@ public SafeUIBlockingUtility(Context context) {
...
     }
 
+    public SafeUIBlockingUtility(Context context, String message) {
+        this.context = context;
+        progressDialog = new ProgressDialog(context);
+        progressDialog.setMessage(message);
+        progressDialog.setCancelable(false);
+        progressDialog.setIndeterminate(true);
+        progressDialog.setTitle(utilityTitle);
+        progressDialog.setProgressStyle(ProgressDialog.STYLE_SPINNER);
```
Comment (luckyman20) [inline mifosng-android/src/main/java/com/mifos/utils/SafeUIBlockingUtility.java]: You can connect all `sets`.

**Comment by OussEmaDevCode** [comment]
@luckyman20 Hi : ), ProgressDialogues are not actual AlertDialogues they just extend the AlertDialogue class so they don't completely work like alertDialogues that's why both progress and alert dialogues have different methods of creation, the alertDialogue lets you use its Builder class to connect all the sets without a problem but unfortunately the progressDialogue doesn't have its own Builder class it just uses the one of the AlertDialogue(that can only return an alertDialogue instance), that's why the correct way of initializing a progressDialogue is by using separate `sets` and that's also the method used in the progressDialogue [class](https://user-images.githubusercontent.com/38397893/70552552-dfec0080-1b79-11ea-8fdb-c98fa60476a1.png).

**Comment by OussEmaDevCode** [comment]
@luckyman20 Done !!

**Comment by OussEmaDevCode** [comment]
@moksh-mahajan Hi : ), please have a look at this.

**Pull request merged**